egs/vaevc/template/run.sh

```diff
@@ -53,7 +54,7 @@ logdir=${datadir}/log; mkdir -p ${logdir}
 if [ "${stage}" -le 0 ] && [ "${stop_stage}" -ge 0 ]; then
     echo "stage 0: download dataset and generate scp"
     ${train_cmd} "${logdir}/download.log" \
-        local/download.sh --wavdir "${wavdir}"
+        local/download.sh "${downloaddir}"
```

Comment:
```
local/download.sh "${downloaddir}"
->
local/download.sh \
  --downloaddir "${downloaddir}"
```
For kaldi-style arguments of shell scripts, it uses parse_options.sh which enables to automatically deploy arguments to named variable. You also need to modify ```local/download.sh```